aws-rds-dbcluster/aws-rds-dbcluster.json

```diff
@@ -178,8 +153,7 @@
     },
     "Port": {
       "description": "The port number on which the instances in the DB cluster accept connections. Default: 3306 if engine is set as aurora or 5432 if set to aurora-postgresql.",
-      "type": "integer",
-      "default": 3306
```

Comment:
Why are we removing the default here?

What will the default be after this change?

Reply:
The engine-specific default value would be set by RDS API